app/models/motion.rb

```diff
@@ -50,6 +50,16 @@ class Motion < ActiveRecord::Base
   scope :closing_in_24_hours,      -> { where('motions.closing_at > ? AND motions.closing_at <= ?', Time.now, 24.hours.from_now) }
   scope :chronologically, -> { order('created_at asc') }
 
+  scope :closing_soon_not_published, -> {
+     voting
+    .joins("LEFT OUTER JOIN events e ON e.eventable_id = motions.id AND e.eventable_type = 'Motion'")
```

Comment:
I think I understand the logic, however I'm wondering there's any chance we could still duplicate the events, or miss the opportunity to publish one. Trying to get my head around the intended functionality. Can you think of any cases that would mean we published too many/too few events?